en/controllers/components/check-http-cache.rst

```diff
@@ -0,0 +1,36 @@
+Checking HTTP Cache
+===================
+
+.. php:class:: CheckHttpCacheComponent(ComponentCollection $collection, array $config = [])
+
+.. versionadded:: 4.4.0
+    The ``CheckHttpCacheComponent`` was added.
+
+The HTTP cache validation model is one of the processes used for cache gateways,
+also known as reverse proxies, to determine if they can serve a stored copy of
+a response to the client. Under this model, you mostly save bandwidth, but when
+used correctly you can also save some CPU processing, reducing this way response
```

Comment:
> reducing this way response times

Is that wording intentional?

Reply:
No, I'm a sloppy writer.

Reply:
Not gonna lie, I googled "this way response" before asking 🤣